### PR TITLE
Template spec solution update for Infoblox Cloud Data Connector

### PR DESCRIPTION
**Pull request by vmanojreddy**
Change(s):

See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Infoblox Cloud Data Connector.

See guidance below
Version Updated:
2.0.2

See guidance below
Testing Completed:
The link for test results is
https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B66FD3D9E-6773-4A10-981E-B22A45D4C82B%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20Infoblox%20Cloud%20Data%20Connector.docx&action=default&mobileredirect=true&share=IQGePf1mc2cQSpgesipF1MgrAZ7MpFpWmUzIQUiOKZW020U

### CODE REVIEW

**Comment by v-amolpatil** [comment]
Please update `Azure Sentinel` to `Microsoft Sentinel` for below files
image

**Comment by v-amolpatil** [comment]
Please verify if we need to update the folder name and title in this solution. In text change document its is as shown below
image

File: Solutions/Infoblox Cloud Data Connector/Data/Solution_Infoblox.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-    "Workbooks": ["Workbooks/InfobloxCDCB1TDWorkbook.json"],
+    "Description": "The [Infoblox cloud](https://www.infoblox.com/) managed Data Connector (DC) is a utility designed to collect DNS query and response data and security logs and transfer the data to defined destinations such as the BloxOne Threat Defense Cloud, Infoblox NIOS reporting server, and syslog servers such as a SIEM (Security Information and Event Manager).**Underlying Microsoft Technologies used:** \r\n \r\n This solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs:\r\n \r\n a. [Agent based logs collection from Windows and Linux machines ](https://docs.microsoft.com/azure/azure-monitor/agents/data-sources-custom-logs)",
```
Comment (v-amolpatil) [inline Solutions/Infoblox Cloud Data Connector/Data/Solution_Infoblox.json:5]: Please update the description as per the text change document.

**Comment by v-amolpatil** [comment]
Please also update the image in test document.

**Comment by vmanojreddy** [comment]
> Please update `Azure Sentinel` to `Microsoft Sentinel` for below files image

Done

File: Solutions/Infoblox Cloud Data Connector/Data/Solution_Infoblox.json (hunk 1 above)
Comment (vmanojreddy) [reply Solutions/Infoblox Cloud Data Connector/Data/Solution_Infoblox.json]: Done

**Comment by v-amolpatil** [comment]
Please update isPreview to false

image

**Review by v-amolpatil — APPROVED** [review]

**Pull request merged**